src/klein/_app.py

```diff
@@ -52,18 +52,38 @@
 ]
 
 
-class KleinRouteHandler(Protocol):
-    def __call__(_self, self: Any, request: IRequest) -> KleinRenderable:
+class KleinRouteFunction(Protocol):
```

Comment:
This is ok and maybe this is self understood for people familiar with typing.

I had to see whether the Protocol is a real Twisted protocol or a typing one.

Maybe explicitly use `typing.Protocol' to make it easier to detect that this is defined only for typing usage.

```suggestion
class KleinRouteFunction(typing.Protocol):
```

Reply:
Interesting… though the reverse may be more appropriate over time; I would expect `typing.Protocol` to be more common in the Klein code base than `twised.internet.Protocol`, since Klein should hopefully not be going that far down the network stack much.

Reply:
You are right.